src/menu.stories.ts

```diff
@@ -86,6 +87,18 @@ const meta: Meta = {
         },
       },
     },
+    loading: {
```

Comment:
Strictly speaking, `loading` should be an attribute on Menu Options because it's what shows the loading state. But I feel it's more natural for consumers for `loading` to be an attribute on Menu. It also aligns Menu with Dropdown. What do you think?

Reply:
Yeah, I agree with where you have it 👍